server/py/framework/db/sqldb/db.py

```diff
@@ -594,9 +602,9 @@ def _update_run_end_time(run: Run, run_dict: dict, end_time: Optional[str] = Non
             and not run.end_time
         ):
             if end_time is None:
-                end_time = datetime.now(timezone.utc)
+                # Ensures fsp 6 for MySQL NOW() to includes microseconds
+                end_time = func.now(6)
```

Comment:
Is this TZ-aware?

Reply:
NOW() follow its system timezone. mysql has utc by default. I dont think it is necessary to explicitly state `utc` - as for consistency with other datetime being generated automatically.

@alonmr can you use kwarg to set the fsp rather than magic number?

Reply:
it is UTC but it doesn't include the tzinfo so I'll add it manually with helpers we have